Fixed symbol table printing

kunjsong01 opened the pull request:
Signed-off-by: kunjsong01 <kunjian90@gmail.com>

This patch fixed symbol table printing. 
## Problem: Since "show_symbol_table()" is an empty function, ESBMC doesn't print the symbol table when giving "--symbol-table-only" or "--symbol-table-too" arguments. 

## Solution: Use "show_symbol_table_plain()" instead.

kunjsong01 committed:
Fixed symbol table printing

Signed-off-by: kunjsong01 <kunjian90@gmail.com>

lucasccordeiro reviewed (approved):
LGTM.

Can you add a test case?

kunjsong01 commented:
For each PR, are we supposed to add the corresponding test case in **esbmc/regression/esbmc**? 
Something like **esbmc/regression/esbmc/github_<PR_number>**?

lucasccordeiro commented:
Yes.

Sent from my iPhone

> On 7 Aug 2021, at 16:26, KSong ***@***.***> wrote:
> 
> ﻿
> For each PR, are we supposed to add the corresponding test case in esbmc/regression/esbmc?
> Something like esbmc/regression/esbmc/github_<PR_number>?
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or unsubscribe.

kunjsong01 committed:
Added test case for PR #485 - Fixed symbol table printing

Signed-off-by: kunjsong01 <kunjian90@gmail.com>

kunjsong01 commented:
Added the corresponding test case in new commit 082de14. 

I copied **esbmc/regression/esbmc/overflow_01_addition** and added the affected argument "--symbol-table-too" in the config file test.desc.

kunjsong01 commented:
Received an email regarding commit ecfbec9: 
'''
Build All Solvers / Build ESBMC with Boolector (ARM64)
Failed in 0 seconds 
'''

Are we also building ESBMC on aarch64 target?

lucasccordeiro commented:
> Are we also building ESBMC on aarch64 target?

@rafaelsamenezes and @pmatos managed to build ESBMC on aarch64 target.

rafaelsamenezes reviewed (approved):
LGTM
